pi5/board.go

```diff
@@ -186,7 +191,7 @@ func (b *pinctrlpi5) setPulls() {
 
 		// only the 5th and 6th bits of the register are used to set pull up/down
 		// reset the register then set the mode
-		b.vPage[PadsBank0Offset+pinOffsetBytes] = (b.vPage[PadsBank0Offset+pinOffsetBytes] & 0xf3) | mode
+		b.boardPinCtrl.VPage[PadsBank0Offset+pinOffsetBytes] = (b.boardPinCtrl.VPage[PadsBank0Offset+pinOffsetBytes] & 0xf3) | mode
```

Comment:
similar thoughts, this feature does not seem generic enough to make a helper for it